### ISSUE TITLE
Show error or validation message when repository ID is wrong when creating a repository in Fabrica

### ISSUE THREAD

**Issue by MaryHirsch** [datacite/bracco#497]
The format for a repository ID should be the member id followed by a dot and then a sting of letters. Currently, it is possible to edit the "prefix" part of the repository ID, but this does not work. However, no error message is shown to the user when they try to create the new repository.

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_2oclx)

**Comment by codycooperross** [comment]
Right—the validation should just be performed on the form entry in bracco since there can be a mismatch between the provider in a repository ID and the repository's actual provider.
